src/shared/components/Footer/FooterList/component.tsx

```diff
@@ -30,46 +21,49 @@ const calculateFlexOrder = (index: number, arrayLength: number): number => {
 };
 
 const FooterList: React.FC = () => {
-  const footerLists: Array<FooterList> = [
+  const footerLists: Array<MenuList> = [
     {
+      id: Math.random(),
       heading: 'Категорії',
       list: [
-        { name: 'Освіта', link: '/' },
-        { name: 'Діти', link: '/' },
-        { name: 'Екологія', link: '/' },
-        { name: 'Війна', link: '/' },
-        { name: 'Медицина', link: '/' },
-        { name: 'Мистецтво', link: '/' },
+        { name: 'Освіта', link: '/', id: Math.random() },
+        { name: 'Діти', link: '/', id: Math.random() },
+        { name: 'Екологія', link: '/', id: Math.random() },
+        { name: 'Війна', link: '/', id: Math.random() },
+        { name: 'Медицина', link: '/', id: Math.random() },
+        { name: 'Мистецтво', link: '/', id: Math.random() },
       ],
     },
     {
+      id: Math.random(),
       heading: 'Про нас',
       list: [
-        { name: 'Як це працює', link: '/' },
-        { name: 'Поради', link: '/' },
-        { name: 'Кейси', link: '/' },
-        { name: 'Контакти', link: '/' },
+        { name: 'Як це працює', link: '/', id: Math.random() },
+        { name: 'Поради', link: '/', id: Math.random() },
+        { name: 'Кейси', link: '/', id: Math.random() },
+        { name: 'Контакти', link: '/', id: Math.random() },
       ],
     },
     {
+      id: Math.random(),
       heading: 'Проект',
       list: [
-        { name: 'Знайти проект', link: '/' },
-        { name: 'Додати проект', link: '/' },
-        { name: 'Критерії відбору проекту', link: '/' },
+        { name: 'Знайти проект', link: '/', id: Math.random() },
+        { name: 'Додати проект', link: '/', id: Math.random() },
+        { name: 'Критерії відбору проекту', link: '/', id: Math.random() },
       ],
     },
   ];
 
   return (
     <FooterListWrapper>
-      {footerLists.map((footerList: FooterList, index: number) => (
-        <div key={index}>
+      {footerLists.map((footerList: MenuList, index: number) => (
+        <div key={footerList.id}>
           <FooterListHeading>{footerList.heading}</FooterListHeading>
           <ul>
-            {footerList.list.map((item: FooterItem, i: number) => (
+            {footerList.list.map((item: MenuItem, i: number) => (
```

Comment:
`i: number` could be removed

Reply:
Isn't that necessary for your order function?

Reply:
right 👍 sorry, missed that 👍